src/app/public/modules/skip-link/skip-link-host.component.scss

```diff
@@ -0,0 +1,19 @@
+.sky-skip-link {
```

Comment:
I think we have a mixin for `sky-screen-reader-only`. Could we reuse that here ?

Reply:
Good eye, but I believe the code for this is slightly different, taken from: https://www.bignerdranch.com/blog/web-accessibility-skip-navigation-links/